core/client/src/block_builder/block_builder.rs

```diff
@@ -104,7 +103,7 @@ where
 				ExecutionContext::BlockConstruction,
 				xt.clone()
 			)? {
-				Ok(ApplyOutcome::Success) | Ok(ApplyOutcome::Fail) => {
+				Ok(_) => {
```

Comment:
Don't we still need to check the value?

Reply:
`ApplyOutcome` have only two variants `Success` and `Fail`. it makes no differences in terms of functionality.
In semantic level, as long as the transaction is applied, it will be pushed and doesn't care about the apply outcome.